masternode-manager/main.ts

```diff
@@ -40,6 +38,28 @@ class App {
       }
     }
   }
+
+  async setupNeededComponents(): Promise<string> {
+    try {
+      await this.communication.connect();
+
+      const ownerWallet: string = await this.communication.query(Operation.RECEIVE_WALLET_NAME);
+      this.logger.info('Connected to wallet', ownerWallet);
+
+      const address: string = await this.communication.query(Operation.RECEIVE_ADDRESS);
+      const signature: string = await this.communication.query(
+        Operation.SIGN_MESSAGE,
```

Comment:
Get the sign message from [API](https://dev.api.lock.space/swagger/#/auth/AuthController_getSignMessage), not from config

Reply:
oh yeah true... will change